### PR TITLE
Rhaas/follow redirect

### PR DESCRIPTION
**Pull request by rhaas80**

### CODE REVIEW

**Comment by rhaas80** [comment]
Ticket is: https://trac.einsteintoolkit.org/ticket/1985

File: GetComponents [hunk 1]
```diff
@@ -1777,12 +1777,12 @@ sub handle_curl {
...
               . "rm -rf $tmpdir && "
               . "mkdir $tmpdir && "
-              . "(cd $tmpdir && $curl -O $user $url/$checkout) && "
+              . "(cd $tmpdir && $curl --location -O $user $url/$checkout) && "
               . "mv $tmpdir/$checkout $target/$name &&"
               . "rmdir $tmpdir; " . "}";
         }
         else {
-            $cmd = "cd $target && $curl -O $user $url/$checkout";
+            $cmd = "cd $target && $curl --location -O $user --location $url/$checkout";
```
Comment (knarrff) [inline GetComponents]: Why is this added twice here?

File: GetComponents [hunk 2]
```diff
@@ -1805,12 +1805,12 @@ sub handle_curl {
...
               . "mkdir $tmpdir && "
               . "mv $name $tmpdir/$checkout && "
-              . "(cd $tmpdir && $curl -O $user $url/$checkout) && "
+              . "(cd $tmpdir && $curl --location -O $user $url/$checkout) && "
               . "mv $tmpdir/$checkout $target/$name &&"
               . "rmdir $tmpdir; " . "}";
         }
         else {
-            $cmd = "cd $target && $curl -O $user $url/$checkout";
+            $cmd = "cd $target && $curl --location -O $user --location $url/$checkout";
```
Comment (knarrff) [inline GetComponents]: Why is this added twice here?
